graphql/src/execution/execution.rs

```diff
@@ -356,27 +354,29 @@ where
             &ctx.schema.document
         },
         type_name,
-    ).ok_or(QueryExecutionError::NamedTypeError(type_name.to_string()))?;
+    ).ok_or(vec![QueryExecutionError::NamedTypeError(
```

Comment:
There is `impl From<QueryExecutionError> for Vec<QueryExecutionError>` so you can remove the surrounding `vec!`